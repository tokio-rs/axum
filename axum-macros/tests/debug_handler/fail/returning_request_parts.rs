#[axum::debug_handler]
async fn handler() -> (
    axum::http::request::Parts, // this should be response parts, not request parts
    axum::http::StatusCode,
) {
    panic!()
}

fn main() {}
