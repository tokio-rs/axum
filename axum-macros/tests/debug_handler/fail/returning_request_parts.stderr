error[E0308]: mismatched types
 --> tests/debug_handler/fail/returning_request_parts.rs:3:5
  |
3 |     axum::http::request::Parts, // this should be response parts, not request parts
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^
  |     |
  |     expected `axum::http::response::Parts`, found `axum::http::request::Parts`
  |     expected `axum::http::response::Parts` because of return type
